static/styles/dark_theme.css

```diff
@@ -519,6 +520,7 @@ body.dark-theme {
     }
 
     .expand_composebox_button:hover,
+    .narrow_to_compose_recipients:hover,
     .collapse_composebox_button:hover,
```

Comment:
3. The color is the same as the others in the top row(expand, close). I wonder if it's because of the thickness that it looks different from the others?